pkg/kubecli/kubecli.go

```diff
@@ -35,14 +37,16 @@ import (
 var (
 	kubeconfig string
 	master     string
+	server     string
 )
 
 func init() {
-	flag.StringVar(&kubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
-	flag.StringVar(&master, "master", "", "master url")
+	flag.StringVar(&kubeconfig, "kubeconfig", "", "Absolute path to the kubeconfig file")
+	flag.StringVar(&master, "master", "", "Deprecated: use server instead")
```

Comment:
we're in alpha. why not 's/master/server'?

Reply:
I'm game if everybody's on board with that.

Reply:
+1 on davids comment. Especially since I think that we should create the client for virtcl the same way like it is created for kubectl, which will have the result that code changes here will only affect our server components.

See https://github.com/kubevirt/kubevirt/pull/648#pullrequestreview-89509254 for details on how to solve it for virtcl.